Build.mak

```diff
@@ -0,0 +1,16 @@
+# Set of flags required to compile vibe.d without dub
+override DFLAGS += -version=VibeLibeventDriver -version=Have_openssl \
+	-I./submodules/vibed/source -I./submodules/libevent -I./submodules/openssl
+
+# Compile all vibe.d as a single static library because rdmd can't track
+# all imported dependencies correctly in this case
+$O/libvibed.a: $(shell find $C/submodules/vibed/source -type f -name '*.d')
```

Comment:
Sorry, I don't know why lately my brain is completely filtering comments. In any case:

1. Why is that? Is that an rdmd bug? Shouldn't it be reported if it is?
2. Why not just adding all vibed files as dependencies to the binary and skip the static library?